auto-completion/fish/buku.fish

```diff
@@ -6,7 +6,7 @@
 #
 complete -c buku -s a -l add     -r --description 'add bookmark'
 complete -c buku -l ai              --description 'auto-import bookmarks'
-complete -c buku -l colors          --description 'set output colors (see man page for details)'
+complete -c buku -l colors          --description 'set output colors'
```

Comment:
Please change to:

complete -c buku -l colors       -r --description 'set output colors'

The `-r` stands for required.